### PR TITLE
RH-SSO Install

### PR DESCRIPTION
**Pull request by pmccarthy**
**Summary**
Adding playbook and role for the installation of RH-SSO service on Openshift

**Validation**
```
oc login https://<public_master_host>:8443
ansible-playbook -i inventories/hosts playbooks/rhsso.yml
```

### CODE REVIEW

**Comment by maleck13** [comment]
@pmccarthy strange error 
``` fatal: [127.0.0.1]: FAILED! => {"changed": false, "msg": "invalid literal for int() with base 10: ''"}```

File: evals/roles/rhsso/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- name: Generate auth token
+  shell: oc whoami -t
+  register: auth_token
+
+- name: "Updating to latest RH-SSO template: {{ rhsso_template }}"
+  shell: oc replace -n openshift --force -f https://raw.githubusercontent.com/jboss-openshift/application-templates/ose-v1.4.11/sso/sso72-x509-postgresql-persistent.json
+  when: rhsso_force_template_update
+
+- name: Install image streams
+  shell: oc -n openshift import-image redhat-sso72-openshift:1.1
```
Comment (odra) [inline evals/roles/rhsso/tasks/main.yml]: Do you think it makes sense to make `redhat-sso72-openshift:1.1` a defaults var instead of hard-coding it?

Comment (pmccarthy) [reply evals/roles/rhsso/tasks/main.yml]: @odra Good point, I'll expose it as a variable instead

File: evals/roles/rhsso/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+- name: Generate auth token
+  shell: oc whoami -t
+  register: auth_token
+
+- name: "Updating to latest RH-SSO template: {{ rhsso_template }}"
+  shell: oc replace -n openshift --force -f https://raw.githubusercontent.com/jboss-openshift/application-templates/ose-v1.4.11/sso/sso72-x509-postgresql-persistent.json
```
Comment (aidenkeating) [inline evals/roles/rhsso/tasks/main.yml]: Would there be a benefit to making this template URL a variable?

Comment (pmccarthy) [reply evals/roles/rhsso/tasks/main.yml]: @aidenkeating Sure, makes sense. Updating now

**Comment by pmccarthy** [comment]
@maleck13 I've made a few updates, hopefully it should work for you now. 

Note: The validation command is now:
```
ansible-playbook -i inventories/hosts playbooks/rhsso.yml
```

**Comment by maleck13** [comment]
@pmccarthy :( got a different error 

```
fatal: [127.0.0.1]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (openshift_raw) module: description Supported parameters include: api_key, api_version, cert_file, context, force, host, key_file, kind, kubeconfig, name, namespace, password, resource_definition, src, ssl_ca_cert, state, username, verify_ssl"}

```

**Pull request merged**